global.json

```diff
@@ -1,9 +1,9 @@
 {
   "sdk": {
-    "version": "5.0.100-rc.1.20429.2"
+    "version": "5.0.100-rc.2.20460.25"
   },
   "tools": {
-    "dotnet": "5.0.100-rc.1.20429.2",
+    "dotnet": "5.0.100-rc.2.20460.25",
```

Comment:
@jeffhandley \ @buyaa-n updating to the most recent (as of yesterday)'s  rc2 build didn't seem to help with https://github.com/dotnet/aspnetcore/blob/release/5.0-rc2/src/Shared/CertificateGeneration/CertificateManager.cs#L30-L36

Reply:
Not sure if the analyzers updates merged to sdk yet, i meant about dogfooding from the Microsoft.CodeAnalysis.NetAnalyzers release cc @mavasani

Reply:
Tagging @jmarolf - we haven't update the analyzers inserted into the SDK for a while: https://github.com/dotnet/sdk/blob/b34cf11e9347c5e32f5d661d3868bd0654a6cb2b/eng/Version.Details.xml#L164-L167

@pranavkm I think a better route for you might be to follow the runtime repo's approach of adding an explicit package reference to `Microsoft.CodeAnalysis.NetAnalyzers` NuGet package instead of waiting on the SDK catching up on latest analyzer bits. https://github.com/dotnet/runtime/blob/23919a4b58d888b92f5192576febf47b06942b2c/eng/Analyzers.props#L9. This package autoamtically turns off analyzers from the SDK to avoid dupes.

Reply:
@dougbu \ @Pilchie  how do you feel about doing this as part of this PR?

Reply:
Unless there is a big need for it, I'd much rather stay on the RC1 build (ideally the RC1 build that will ship) to make it easier for people to have the right SDK.

Reply:
```suggestion
    "dotnet": "5.0.100-rc.1.20452.10",
```

Reply:
@Pilchie due to our requirements for added runtimes, users can't use a global .NET installation in this repo. That said, I'm not sure we have a real reason to move to a newer SDK given the `Microsoft.CodeAnalysis.NetAnalyzers` option.

side note: We removed some of the `Microsoft.CodeAnalysis.*` packages because they caused weird coherency problems in dependency updates into aspnetcore-tooling. If those don't happen again, party on w/ `Microsoft.CodeAnalysis.NetAnalyzers`.